cmd/bootstrap/runner.go

```diff
@@ -354,9 +358,6 @@ func (r *runner) installCatalogs(ctx context.Context, k8sClients k8sclient.Inter
 			ObjectMeta: metav1.ObjectMeta{
 				Name:      name,
 				Namespace: metav1.NamespaceDefault,
-				Labels: map[string]string{
-					label.CatalogVisibility: "internal",
-				},
```

Comment:
Not using `helm-stable` index any more. The new index ist not compatible with being treated as `internal` catalog because the archive urls are not as stable as Giant Swarm ones